lib/twofishes/result.rb

```diff
@@ -1,56 +1,32 @@
-require 'rash'
-
+require 'delegate'
 module Twofishes
-
-  class Result
-
-    def initialize(hash)
-      @data = Hashie::Rash.new(hash)
-    end
-
-    def self.from_response(hash)
-      if hash['interpretations']
-        hash['interpretations'].map { |interpretation| self.new(interpretation) }
-      else
-        []
-      end
-    end
-
-    def name
-      feature.name
-    end
-
-    def display_name
-      feature.display_name
-    end
-
-    def country_code
-      feature.cc
-    end
-
-    def lat
-      feature.geometry.center.lat
-    end
-
-    def lng
-      feature.geometry.center.lng
+  class Result < SimpleDelegator
+    def self.from_response(response)
+      response.interpretations.map { |interpretation|
+        new(interpretation)
+      }
     end
+  end
+end
 
-    def coordinates
-      [lat, lng]
-    end
+require 'forwardable'
```

Comment:
I think it kinda belongs here since it alters Result and it's a monkeypatch. Open to suggestions.

And yes, it's a thrift generated class.